config/eslint/eslint.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Has to be JSON due to `"type": "module"` in the package.json causing `eslint` to be unable to read the config since it only understands commonJS.